tests/components/google_travel_time/test_config_flow.py

```diff
@@ -36,13 +36,18 @@ async def assert_common_reconfigure_steps(
     hass: HomeAssistant, reconfigure_result: config_entries.ConfigFlowResult
 ) -> None:
     """Step through and assert the happy case reconfigure flow."""
+    client_mock = AsyncMock()
     with (
-        patch("homeassistant.components.google_travel_time.helpers.Client"),
         patch(
-            "homeassistant.components.google_travel_time.helpers.distance_matrix",
-            return_value=None,
+            "homeassistant.components.google_travel_time.helpers.RoutesAsyncClient",
+            return_value=client_mock,
+        ),
+        patch(
+            "homeassistant.components.google_travel_time.sensor.RoutesAsyncClient",
+            return_value=client_mock,
         ),
     ):
+        client_mock.compute_routes.return_value = None
```

Comment:
why dont you use the other mock here

Reply:
But mocks are created for every test, so they are reset automatically. Check an integration like `airgradient` or `spotify` for an example. I just patch the mock with a good default value that works for a ton of tests, and then I update the mock where needed